manifests/daemonset_openshift.yaml

```diff
@@ -0,0 +1,169 @@
+        securityContext:
+          privileged: true
+        volumeMounts:
```

Comment:
I find it strange that you do not need to use a particular SCC where you explicitly allow volume mounts of `hostPath` type. Likewise for allowing privileged pods.

Reply:
> Maybe due to kube-system namespace, chmm

Indeed, looks like that's the reason. After changing the namespace to `bridge-filtering-cni`, I had to[ add a SCC](https://github.com/rhrazdil/bridge-filtering-cni/pull/12/commits/4419df4b7280922fe3ab498ef6ff7eb053690967)